### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by vjuranek**

### CODE REVIEW

File: documentation/README.md [hunk 1]
```diff
@@ -1,16 +1,13 @@
 # Documentation
 
 The documentation in this repository is written along the same guidelines as core Infinispan documentation,
-more of which can be learned about [here](https://github.com/infinispan/infinispan/blob/5.3.x/documentation/README.md).
+more of which can be learned about [here](https://github.com/infinispan/infinispan/blob/master/documentation/README.md).
 
-This book is incorporated in the [Infinispan Documentation Home](http://infinispan.org/documentation)
+This book is available on the [Infinispan Cloud Cache Stores documentation page](http://infinispan.org/docs/cachestores/cloud/).
 
 ## Differences from the core Infinispan documentation
```
Comment (tristantarrant) [inline documentation/README.md]: I think this entire paragraph should be removed

Comment (vjuranek) [inline documentation/README.md]: done, PR updated

File: documentation/README.md [hunk 2]
```diff
@@ -1,16 +1,11 @@
...
-This book is incorporated in the [Infinispan Documentation Home](http://infinispan.org/documentation)
+more of which can be learned about [here](https://github.com/infinispan/infinispan/blob/master/documentation/README.md).
 
 ## Differences from the core Infinispan documentation
 
-Documentation written for Hot Rod clients should be in a single AsciiDoc
-document titled `index.adoc` and stored in `src/main/asciidoc`
-You may use tables of contents and various other AsciiDoc features as 
-described in the [document authoring guidelines](http://infinispan.org/docs/6.0.x/contributing/contributing.html#_writing_documentation_and_faqs).
+Documentation written should be in a single AsciiDoc document titled `index.adoc` and stored in `src/main/asciidoc`.
```
Comment (tristantarrant) [inline documentation/README.md]: Please remove this whole paragraph

File: documentation/README.md [hunk 3]
```diff
@@ -1,16 +1,11 @@
...
+You may use tables of contents and various other AsciiDoc features as described in the [document authoring guidelines](http://infinispan.org/docs/stable/contributing/contributing.html#writing_documentation_and_faqs).
 
-All other aspects of the [document authoring guidelines](http://infinispan.org/docs/6.0.x/contributing/contributing.html#_writing_documentation_and_faqs)
-must be adhered to, particularly sections on style and voice.
\ No newline at end of file
+All other aspects of the [document authoring guidelines](http://infinispan.org/docs/stable/contributing/contributing.html#writing_documentation_and_faqs) must be adhered to, particularly sections on style and voice.
```
Comment (tristantarrant) [inline documentation/README.md]: Please remove this whole paragraph

**Comment by vjuranek** [comment]
paragraphs are removed now

**Pull request merged**